deploy-manage/upgrade/deployment-or-cluster/reading-indices-from-older-elasticsearch-versions.md

```diff
@@ -6,6 +6,8 @@ applies_to:
   deployment:
     ess:
     self:
+    eck:
+    ece:
```

Comment:
same here - page might need to be moved

Reply:
Created https://github.com/elastic/docs-content/issues/2343